test/Driver/filelists.swift

```diff
@@ -5,15 +5,28 @@
 
 // CHECK-NOT: Handled
 // CHECK: Handled a.swift
+// CHECK-NEXT: Supplementary a.swift:
+// CHECK-NEXT: Supplementary a.swiftdoc
+// CHECK-NEXT: Supplementary a.swiftmodule
```

Comment:
Shouldn't this be something like "Supplementary swiftdoc: a.swiftdoc"?

Reply:
Good point! I'll write more Python.